pkg/checks/external/checkclient/main.go

```diff
@@ -16,6 +16,11 @@ import (
 	"github.com/Comcast/kuberhealthy/v2/pkg/checks/external/status"
 )
 
+const (
+	podAPIVersion = "v1"
+	podKind       = "Pod"
+)
+
```

Comment:
is there a reason this is needed here? :)

Reply:
Nope, me that forgot to clean them after I moved the function. 
Thanks for spotting it.